### ISSUE TITLE
[Bug Report][4.2.1] Selectable VList should have a listbox role

### ISSUE THREAD

**Issue by brendan-nijboer-bw** [vuetifyjs/vuetify#23174]
### Environment
**Vuetify Version:** 4.2.1
**Vue Version:** 3.5.25
**OS:** Linux undefined (current)

### Expected Behavior

Since the VListItems within a selectable VList have an `option` role, the containing VList should have a `listbox` role as specified in [the MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/option_role)

### Actual Behavior

The VList has a `list` role.

### Reproduction Link

[Reproduction link](https://play.vuetifyjs.com/#eNqlVV1v2zYU/Suc+iAZs6mkblDMSLoMQ4cV2MOwFn2J86BYVzZbitRI2q1h+L/38EOy4gZFgAIWTN57ee65H7y8O2Si7bRxs7bq+CerVbbIDkvF2DIp7DJbsCDxst2W/H6ZbZzr7KIsV7XCsZqk2BmuyJWqa8tbmJVmq5xoaVbr9nbOr/jLq7IW1o3lnGw7ezD6iyUDlGU2HfkpIdyRmRlSNRkyz/V7duyR7zPdD/070eyf4TNZ3r7iL/ll9JJEM1k9WO8iQHvk41Lhl00zKdRne5bulQ2pvusp/JTXAOaR7uESDq1ZlX90HYcRPF47ajtZOXrjTa53s6rrwjJsVlq5SigySRSEEi7YbtZqEFlYkrRyVN8ssyD4WEnfF4P9cGIm4AnHGm1gq5hQbL7M2OIz7f3eL51wkrDJ33nTnP3KgnjnIZNNeSJSRlwIetGJANr0MNqyI3LdH3ock5fEkK/LUSqwtSsjOscsuW3MSLwE7MAMNezIGqNBElnMIwPgIi8jpzfesFBbKScePeIBKdWgBQnuLGowAK8MwT2KcwY/GKTCDmrel/pk5E/3ylTmkxKyEE+gHAkjeDAdPBf4QBdSDn0j1pyM0ebvStWSDCwR15sQq5YUdQXhQDyytVQkRoOs1bjlRf4C6xzCFH2gEcM31OodvXd7SR+qdQGbf8FfWOKVlMWd1FX9F2pWTKbMr99558XkPjhotmrlhFZI9RiEFZM4qGKMDpIbVuvVtiXl+P9bMvv3oXHBPn/RXxe3IQwj6zHshsh5uih7w4pfgDCBD7c1odbY8q7C7HBvJc4AM/r/cyNkXXjjcL1H/DzzwC4gs2KD3kgcIyxT9IWlwIvCEPK7oymUn0Bz4pOeJm8MyY+NcUyxgIlNkXt1pM+CKfhJmOej4EZKTwZa/zeSVnX9dge0f3DLCPcFqAgi95wCuTH897ahNYJxCCDaDnQfdL3n6AhM38IDBP3xyaz52g/1TLl6nM0i7wdkA1vL11qvJVWdwPDTbYkB+HtTtULub/7TD9rpxeXFxXSO7xW+K3yv8f12ceET9r3/0G/PJfD0W9TWIlC7fc2/ejq4+46MqGRNVqyV8B54i3EAXSKRrklqzSfnxMezYXAaBUtFX4Np7JV+avTXPB0sQkCh6Ydnvaam2kr3IQrRRuuNy9NTWJahSNj5QmXHaRbfU1Cd88s5n2fTppKWHj8xvSyRyKaHYbnIwpuVHe+/AfjO6Cw=)

### Images
![https://cdn.cosmicjs.com/23b47e80-ad01-11f1-be84-89054212689f-Screenshot-from-2026-09-10-12-16-35.png](https://cdn.cosmicjs.com/23b47e80-ad01-11f1-be84-89054212689f-Screenshot-from-2026-09-10-12-16-35.png)

**Comment by J-Sek** [comment]
requires `selectable` or `activatable` prop
